src/main/java/com/agata/jeeshop/controllers/RestOrderController.java

```diff
@@ -0,0 +1,25 @@
+package com.agata.jeeshop.controllers;
+
+import com.agata.jeeshop.dto.CartDto;
+import com.agata.jeeshop.models.OrderedCart;
+import com.agata.jeeshop.services.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/rest/order")
+public class RestOrderController {
+    @Autowired
+    private CartService cartService;
+
+    @PostMapping
+    public ResponseEntity<OrderedCart> submitOrder(@RequestBody OrderedCart orderedCart){
```

Comment:
Przekazujesz tutaj OrderdCart, ale zrobiłaś nad tą klasą adnotację @Entity czyli powiedziałaś, że ma to być obiekt tożsamy z bazą danych. Tutaj powinnaś użyć dto (o takiej samej strukturze)